### ISSUE TITLE
[ ACS - Plush: Kitsune 🦊 ] *Yawn*

### ISSUE THREAD

**Issue by Professor-JMO** [Monika-After-Story/MonikaModDev#4078]
ʃ i m o n's Sowing Projects
                                      Format: Unicode
                                        Code: #3174 
                                         04.11.2019
📁⯈ get_some_rest
---
⯆ Open: _"Comfort.rtf"_

---

⮞ ```¿ "There's nothing quite like a peaceful rest after a busy day." ```
⮞ ```m "I can only imagine what it would be like for real...to be right there with you..."```
⤷ ```m "Being wrapped in your arms..."```

>>>>![Stable DDLC ( Preview )](https://user-images.githubusercontent.com/43480604/55995986-acb84100-5c7b-11e9-90a7-8207c61f42db.png)

⮞ A Kitsune plushy!
⤷ Should you feel lonely or fussy.. 🧡

⮞ After a certain _Tanuki_ was released..
⤷ It was only a matter of time until this came along!

⮞ Since when did anyone ever have just one plushy friend anyway?
⤷ They might get lonely!
> _**Who** would want that?_

---

>>>>![Stable Sayori (  preview )](https://user-images.githubusercontent.com/43480604/55995607-91006b00-5c7a-11e9-9aba-5bbeff4563eb.png)

⮞ To the plush!

![Woke](https://user-images.githubusercontent.com/43480604/55997059-0837fe00-5c7f-11e9-85e2-fb67b324bdd0.PNG)

⮞ With [ 2 ] variations!

• It looks like a Fox.
⤷ You look like a _**Fox**_.
>I'm sorry, I didn't mean that.
>>B u t
>>>They are cute are they not?

• Framework ready?
⤷ Yeah, sort of.

• Is that a new hat for Monika?
⤷ I mean...not _**Intentionally**_ it isn't.
>B u t that would probably look super adorable~
>>Probably makes a better _pillow_ if anything.

• How can I use this?
⤷ You **can't**
> The item is currently stuck in Limbo.
>> Only time and circumstance will tell.

⮞ _Dejavu_.
>⮞ _"Psychologists say stuffed animals can be a transitional (or a security) object. It turns out people can feel a bond with objects even if they can’t quite respond to you. Although any true fan of stuffed animals will tell you stuffed animals have feelings."_ 

⮞ If you browse GitHub regularly.
⤷ [Stylish: Custom web page themes](https://userstyles.org/styles/browse?search_terms=github&type=false)
> Dark looks so much better here. 👌

⯅ Close

---

📁⯈ stitchings_n_stuff
---
⯆ Open: "plushies"

---

📁⯈ Kitsune Plush

⯆ Open: "kitsuneplush.rtf"

>>>>![Stable Yuri ( Preview )](https://user-images.githubusercontent.com/43480604/55996159-32d48780-5c7c-11e9-85df-0b1e89517632.png)

>⮞V29uZGVyIGlmIHNoZSdsbCBldmVyIGFzayB3aGF0IG91ciBmYXZvcml0ZSBhbmltYWwgaXMuLj8=
![Nap](https://user-images.githubusercontent.com/43480604/55997512-5c8fad80-5c80-11e9-86b4-224948920873.PNG)

General Information
---

⮞ Some stories depict Kitsune as intelligent beings and as possessing paranormal abilities that increase with their age and wisdom. 
⤷ According to Yōkai folklore, all foxes have the ability to shapeshift into human form. While some folktales speak of kitsune employing this ability to trick others—as foxes in folklore often do—other stories portray them as faithful guardians, friends, lovers, and wives.
>Kitsune have become closely associated with Inari, a Shinto kami or spirit, and serve as its messengers. 
>> Because of their potential power and influence, some people make sacrifices to them as to a deity. 

⮞ Kitsune are believed to possess superior intelligence, long life, and magical powers. The word kitsune is often translated as fox spirit. However, this does not mean that kitsune are _ghosts_, nor that they are fundamentally different from regular foxes. 
⤷ Because the word spirit is used to reflect a state of knowledge or enlightenment.
>The zenko (善狐-literally good foxes) are benevolent, celestial foxes associated with Inari; they are sometimes simply called Inari foxes
>On the other hand, the yako (野狐, literally field foxes, also called nogitsune) tend to be mischievous or even malicious.
>> A _ninko_ is an invisible fox spirit that human beings can only perceive when it _possesses_ them. 

⮞ Kitsune are often presented as tricksters, with motives that vary from mischief to malevolence.
⤷ Kitsune keep their promises and strive to repay any favor. Occasionally a kitsune attaches itself to a person or household, where they can cause all sorts of mischief. 
> In one story from the 12th century, only the homeowner's threat to exterminate the foxes convinces them to behave.

⯅ Close

---

📁⯈ contents_n_stuff
---
⯆ Open: _"Sayonara.rtf"_

---

⮞ This took longer then it should up and I'm sleepy!
⤷ _But before I go.._

⮞ I'm always open to requests from the community.
⤷ _I just can't promise I'll get around to them._

• #3174 

⯅ Close

---

📁⯈ the_stuffs
---
⯆ Open: 📎 Attachments 📎

---
>>>>![Stable Natsuki ( Preview )](https://user-images.githubusercontent.com/43480604/55996266-7fb85e00-5c7c-11e9-9dfa-f957b8271151.png)

Meme of the Theme!
---

>>>>![Those thighs](https://user-images.githubusercontent.com/43480604/55998303-58b15a80-5c83-11e9-8a04-631ef8748a4b.png)

---

⯈ [Kitsune 🦊 ](https://github.com/Monika-After-Story/MonikaModDev/files/3070924/Manifest.zip) ⯇ ← 🎩 ?

---

```
from subprocess import call
```
>def play_wave_file([MUGEN - Pillows.wav):](https://www.youtube.com/watch?v=nayS4K7TQ9M&list=PLDO4ENbmp6gWWa-e_CPjmdfYibY2tL5UY&index=12)
      >return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      ># return_code = call(["aplay", "../success.wav"])
      >print "Played successfully" if return_code == 0 else "Play ended in error"

**Comment by Rai99** [comment]
Are you saying that Monika is a fox?

If you take requests, how about a bunny plush? One with droopy ears would be nice I think. 
![Untitled1144](https://user-images.githubusercontent.com/38822045/56002412-4984d980-5c90-11e9-8968-d15c7177b30c.PNG)

![Untitled1145](https://user-images.githubusercontent.com/38822045/56002420-4ee22400-5c90-11e9-9850-8580c0a0fdb4.PNG)

![Untitled1146](https://user-images.githubusercontent.com/38822045/56002526-cf088980-5c90-11e9-84a8-91dcb2f08c17.PNG)
~~Also, those aren't the only pillows to use.~~

**Comment by Professor-JMO** [comment]
> 
> 
> Are you saying that Monika is a fox?

Nope! 
What gave you that impression?? 👀
![Hoohah!](https://user-images.githubusercontent.com/43480604/56050601-ef2c5d00-5d11-11e9-8889-d083b71c4740.jpg)


> If you take requests, how about a bunny plush? One with droopy ears would be nice I think.
Okie Doki!

_Adds to the list of things To-Do._

> ~~Also, those aren't the only pillows to use.~~

~~_Oh? Aha~_~~

**Comment by AsuraShun** [comment]
You forgot about the tails bit

**Comment by AsuraShun** [comment]
That they mature and get stronger with each tail

**Comment by Professor-JMO** [comment]
[ The _Seams Noisy_ update. ]
@Trilasent for Noise suggestion.
@Orcaramelo for Seam suggestion.

**Comment by Rai99** [comment]
![Untitled1147](https://user-images.githubusercontent.com/38822045/56085891-8e884780-5e19-11e9-94bb-a2b8ae442c89.png)

**Comment by Professor-JMO** [comment]
^ I've said it once before and I'll say it again.
> Lilmoni needs some _**supervision**_. 👁️

**Comment by Professor-JMO** [comment]
@Rai99 
Besides the ears, is there anything you'd like to see come out of this project?
> The pose for example.
>> Do you want it similar to the references you've posted or just whatever?

**Comment by Rai99** [comment]
For color, I'd like it to be either white or light brown. 

For pose, I'd like a blob: 
![Untitled1154](https://user-images.githubusercontent.com/38822045/57031584-0de98980-6c16-11e9-8071-17a993a3ee08.PNG)

![Untitled1156](https://user-images.githubusercontent.com/38822045/57031628-31accf80-6c16-11e9-8294-55dcd42c5610.png)

![Untitled1157](https://user-images.githubusercontent.com/38822045/57031643-396c7400-6c16-11e9-8623-c933a077c7da.PNG)

![Untitled1155](https://user-images.githubusercontent.com/38822045/57031744-86e8e100-6c16-11e9-9f4c-2a43b5b37652.PNG)

Other little things, I'd like the paws to be just slightly visible. I think that should be enough for an idea of what I have in mind. If need be, I'm sure tiny other things could be ironed out after we see what you make of it.

**Comment by Professor-JMO** [comment]
Style:
> -More realistic?
> -Something cute?
> -or maybe... _ahem_, Kawaii? 0w0

**Comment by Rai99** [comment]
I'd probably settle with cute. I don't think I'm ready for kawaii just yet. 

Are there going to be a couple variations like with the eyes you had on the kitsune?

~~Bonus points if Monika can put it on her head or if it can balance things on its head or both.~~

**Comment by Professor-JMO** [comment]
Okie Doki!

🤐

**Comment by Professor-JMO** [comment]
@Rai99 
[ Status Update: Concept stages ]

Eye color or gender preference?

**Comment by Rai99** [comment]
I'm fine with black eyes like the tanuki plush you made. 

I don't think gender would make much of a difference, but I guess male.

**Comment by Professor-JMO** [comment]
@Rai99 
[ Update: Review ]

Sorry for the wait.
Could you please review the plush to see if it's to your preference?
⯈ [Plush Bunny Draft 🐇 ](https://github.com/Monika-After-Story/MonikaModDev/files/3268945/Plush.Bunny.Draft.zip) ⯇

**Comment by Rai99** [comment]
- I would personally prefer the bunny to be one pure color. 
Maybe you could make 2 versions; One white and one light brown. 

- I like the sleepy teary eyes, happy eyes and normal (void) eyes. 

- I like the cheeks, but I think I'd prefer no cheeks. 

- I like the idea of being affected by Monika's shadow, but would it be same shadow for all of Monika's poses? I just thought that maybe it would be a bit different in her lean pose and I don't want it too be too much of a hassle. 

- I'm not a fan of the spots personally. 

- I'm not too sure about the noise, but I think I prefer no noise. ~~I'm a quiet person~~

- As for hats, anything can be a hat. 
![Untitled1283](https://user-images.githubusercontent.com/38822045/59159944-a3422e00-8a9e-11e9-835d-ee75dbcce647.PNG) ![tSXgrD5](https://user-images.githubusercontent.com/38822045/59159950-ab01d280-8a9e-11e9-931a-cb263b1a89f6.jpg)
![Untitled1284](https://user-images.githubusercontent.com/38822045/59160008-472bd980-8a9f-11e9-9f04-17ecb178a51d.PNG) ![Untitled1285](https://user-images.githubusercontent.com/38822045/59160009-48f59d00-8a9f-11e9-95c2-3e610cf88252.PNG)

**Comment by Professor-JMO** [comment]
• Okay, I'll play with the concept of [ 2 ] pure colors.
- Is the `light-brown` a good color choice?

• Great~!

• I like the cheeks too.
- I'll keep 'em as an _alternative_ or for some other possible implementation.

• I like the idea of the shadow in the way of consistency b u t if it applies to this plushy for the sake of _consistency_ than it would only make sense for it to apply to the others such as the `Quetzal` too.
- So this raises further questions about whether or not to implement the shadow and if so to the others as well?
-- Theoretically yes, the shadow would shift and alter as Moni herself _shifted_ and _altered_ raises even further questions as to how much to account for this on both Programming and Artistic sides.
> Things to also consider is if "consistency" is the concern than what would happen if Monika was to leave her seat for the moment that she sometimes does? In that event there should be no shadow.
>> I'm probably going to propose some questions in the near future concerning this in discord.

• Okay!
- So no spots? You mentioned one "pure" color so I'm under the impression that excludes spots entirely.

• No noise is fine.
- The whole purpose behind the "noise" is to give the plush the `texture` that it sometimes needs, else it might look less like a plush.

• _Anything_.

**Comment by Rai99** [comment]
I think light brown is a good color choice. 

I do like the noise and I do agree that it gives it that plush look, but I also appreciate some plushies that are so soft and silky smooth, it has almost no visible texture. I might be a bit undecided on the noise.

**Comment by Professor-JMO** [comment]
@Rai99 
I don't know if I'd call this _pure_...
B u t
How the draft for these [ 2 ] colors?

![Bun Ref 1](https://user-images.githubusercontent.com/43480604/59882476-f312d600-9377-11e9-88a5-7bcc09c373ee.PNG)
![Bun Ref 2](https://user-images.githubusercontent.com/43480604/59882480-f4440300-9377-11e9-907f-6dd3b9fe42f5.PNG)

**Comment by Rai99** [comment]
I wasn't expecting there to be patches on the face. I was more expecting it to be one solid color with shading where appropriate. 

For the brown, I think I would at least prefer the darker shade, or maybe the lighter shade is too light for me. 

I like the shades of white you chose, especially the lighter shade, but I'm not too sure on the 2 different shades on different sections. 

For the ears if possible, I'd also like them to be a little sharper and pointier on the ends and for them to be protruding from closer to the top center of the head. I would also prefer if the ears didn't have a patch of different color on the ends. 

I might also have a more specific color/detail request, but that'll be extra for later if you decide to keep working on it.

**Comment by Professor-JMO** [comment]
Thanks for the feedback.
I'll see what changes I can make to it!

**Comment by LordBaaa** [comment]
@Professor-JMO Do you have and could I get the files for this sleep sprite pls?
![55997512-5c8fad80-5c80-11e9-86b4-224948920873](https://user-images.githubusercontent.com/48495287/59956828-e3bb8780-9460-11e9-8dd2-a7061d3c75bd.PNG)
All I can find is this one in the sleep-mode branch and but I was trying to match yours because its so cute!
![torso-leaning-sleep-def](https://user-images.githubusercontent.com/48495287/59956829-e5854b00-9460-11e9-87cd-080b73d4c72b.png)

**Comment by Professor-JMO** [comment]
Hm?
Do you need _her_ in this pose for something, @LordBaaa ?

**Comment by LordBaaa** [comment]
Well I am working on a few submods for my MAS atm. I plan to make my own sleep-mode script since that branch hasn't been updated in 9 months. I like the one I see in your picture better then the one in that branch and I was have problems getting the face to line up properly to the base. When I was asking about issue #4303 it I remembered this pose.

**Comment by Professor-JMO** [comment]
_Pssst...._ @Rai99 
Are these changes about what you were describing?
![White_Bun](https://user-images.githubusercontent.com/43480604/60626183-98ce3800-9daf-11e9-991f-5044715b6024.PNG)

**Comment by Rai99** [comment]
Yeah, I like that better, but I'd like the ears just a little bit higher up and the ends a little bit round. Like, not pointy, but not spherical. Just a very slight rounded edge.

**Comment by Professor-JMO** [comment]
@Rai99 ...?
![White_Bun_Redo_Take 2](https://user-images.githubusercontent.com/43480604/60690239-5b3edd00-9e8b-11e9-8fed-eea79c64306d.PNG)
![White_BlushBun_Redo_Take 2](https://user-images.githubusercontent.com/43480604/60690240-5b3edd00-9e8b-11e9-85e8-9bcef3a4072c.PNG)

**Comment by Rai99** [comment]
Yeah I think that looks good and I like how the bottom one looks.

**Comment by Professor-JMO** [comment]
Excellent!
I'll add the final changes and send it here for it's final review.
You can decide then if you're satisfied with the final concept!

**Comment by Professor-JMO** [comment]
@Rai99 
S u p e r sorry for the wait!
I had to get some last minute feedback in the CAG.

_Did you decide if you prefered the **Noise** texture_?
![White_Bun_Noise](https://user-images.githubusercontent.com/43480604/61559275-eb098d00-aa2e-11e9-8885-b2016d527322.PNG)

**Comment by Rai99** [comment]
I think I'll take it with the noise texture.

**Comment by Professor-JMO** [comment]
@Rai99 , `Top Hat` good Sir/Madam?
🎩
🧐

![Biz_Bun](https://user-images.githubusercontent.com/43480604/61584087-d77e2500-ab07-11e9-93da-584823793ada.PNG)

**Comment by Professor-JMO** [comment]
`Lilpumpkin`.
_Preparing to Manifest_.

![lilpumpkin](https://user-images.githubusercontent.com/43480604/61668839-a5092f00-aca3-11e9-8b94-2cfd453ea196.PNG)

**Comment by Orcaramelo** [comment]
>>>>.Lilpumpkin
THATS THE MOST ADORABLE THING IVE SEEN TODAY
